### ISSUE TITLE
Complete scam

### ISSUE THREAD

**Issue by Eric-Kosovec** [intel/intel-sdsi#2]
Only a matter of time before you bring this to end user CPUs and we have to pay monthly to overclock or even pay per months of usage. You should be ashamed of yourselves as developers.

**Comment by dpmabo** [comment]
Agree. Intel opened the Pandora's box. We will see Xeon365, xxxLake365 soon. Shame on you Intel.

**Comment by zorrobyte** [comment]
This is why I migrated my personal and work systems to AMD.

**Comment by Foosec** [comment]
Yeah but AMD allows locking of CPUs to that specific motherboard.. Better than this but yeesh, what have we come to.

Looks like we really will own *nothing*

**Comment by a1batross** [comment]
AMD is not your bro either. Give them a bit more time to roll out scam defined silicon competitor
